articles/machine-learning/cortana-analytics-playbook-vehicle-telemetry-powerbi.md

```diff
@@ -49,41 +49,23 @@ A vehicle telematics simulator is included as part of this solution. It emits di
 The Vehicle Catalog is a reference dataset containing VIN to model mapping
 
 ## Power BI Dashboard Preparation
-### Deployment
-Once the deployment is completed, you should see the following diagram with all of these components marked in GREEN. 
-
-* To navigate to the corresponding services to validate whether all of these have deployed successfully, click the arrow on the upper right of the green nodes.
-* To download the data simulator package, click the arrow on the upper right on the **Vehicle Telematics Simulator** node. Save and extract the files locally on your machine. 
-
-![Deployed components](./media/cortana-analytics-playbook-vehicle-telemetry-powerbi-dashboard/1-deployed-components.png)
-
-Now, you are ready to configure the Power BI dashboard with rich visualizations to gain real-time and predictive insights on vehicle health and driving habits. It takes about 45 minutes to an hour to create all the reports and configure the dashboard. 
-
 ### Setup Power BI Real-Time Dashboard
-**Generate simulated data**
-
-1. On your local machine, go to the folder where you extracted the Vehicle Telematics Simulator package.
-  ![Simulator folder](./media/cortana-analytics-playbook-vehicle-telemetry-powerbi-dashboard/2-vehicle-telematics-simulator-folder.png)
-2. Execute the application ***CarEventGenerator.exe***.
-3. It emits diagnostic information and signals corresponding to the state of the vehicle and driving pattern at a given point in time. This is published to an Azure Event Hub instance that is configured as part of your deployment.
-
-![Diagnostics](./media/cortana-analytics-playbook-vehicle-telemetry-powerbi-dashboard/3-vehicle-telematics-diagnostics.png)
 
 **Start the real-time dashboard application**
+Once the deployment is completed, you should follow the Manual Operation Instructions
 
-The solution includes an application that generates a real-time dashboard in Power BI. This application listens to an Event Hub instance, from which Stream Analytics publishes the events continuously. For every event that this application receives, it processes the data using a Machine Learning Request-Response scoring endpoint. The resultant dataset is published to the Power BI push APIs for visualization. 
+* Download real-time dashboard application RealtimeDashboardApp.zip, and unzip it.
+*  In the unzipped folder, open app config file 'RealtimeDashboardApp.exe.config', replace appSettings for Eventhub and ML service connection with the values in the Manual Operation Instructions, and save your changes.
```

Comment:
"replace appSettings for Eventhub and ML service connection" ==> "replace appSettings for Eventhub, Blob Storage and ML service connection"